cli/ssh.go

```diff
@@ -224,13 +236,17 @@ func ssh() *cobra.Command {
 
 			sshSession.Stdin = cmd.InOrStdin()
 			sshSession.Stdout = cmd.OutOrStdout()
-			sshSession.Stderr = cmd.OutOrStdout()
+			sshSession.Stderr = cmd.ErrOrStderr()
```

Comment:
This is a drive-by change I did. It seemed wrong but perhaps I didn't understand the purpose?

Reply:
Seems reasonable to me!